### DISCUSSION TITLE
missing settings

### DISCUSSION THREAD

**Discussion by drakon660** [tye-v2/tye2#31] [General]
This is what I see, in tye2 I don't see settings.
<img width="1716" alt="image" src="https://github.com/tye-v2/tye2/assets/3002778/810bc279-e5e2-4976-8825-2fd5f71a1f4c">

**Comment by razvangoga** [comment]
i'm seeing the same header
maybe it's because the docs workflow did not succeed : https://github.com/tye-v2/tye2/actions/runs/8158288818/job/22299876776

**Reply by drakon660** [reply]
you don't see Settings tab in tye2 repo?

**Reply by razvangoga** [reply]
yeah i can.. sorry... was doing 2 things at once... can you try again? i've tweaked the access rights some more

**Reply by drakon660** [reply]
Now it is better. Checking why pipeline is not working.

**Comment by drakon660** [comment]
I am closing this
